pkg/printers/printers.go

```diff
@@ -109,7 +109,16 @@ func GetStandardPrinter(typer runtime.ObjectTyper, encoder runtime.Encoder, deco
 		fallthrough
 	case "":
 
-		printer = NewHumanReadablePrinter(encoder, decoders[0], options)
+		humanPrintFlags := NewHumanPrintFlags()
+		humanPrinter, matched, err := humanPrintFlags.ToPrinter(format, encoder, decoders[0])
```

Comment:
I dislike the fact that we depend on an encoder / decoder on this printer.
One case I saw it used for was when dealing with unstructured objects. Maybe require that those be taken care of by the consumer?